packages/compass-sidebar/src/components/navigation-items.spec.tsx

```diff
@@ -49,7 +49,9 @@ describe('NavigationItems [Component]', function () {
         showCreateDatabaseGuideCue: true,
       });
 
-      await waitFor(() => screen.getByRole('dialog'));
+      await waitFor(() => screen.getByRole('dialog'), {
+        timeout: 2000,
+      });
```

Comment:
I think this is literally what `findBy*` selectors are doing

Reply:
That's absolutely correct! Changing it to that :)